src/build.rs

```diff
@@ -175,7 +175,8 @@ impl BuildDirectory {
         self.workspace.builds_dir().join(&self.name)
     }
 
-    fn source_dir(&self) -> PathBuf {
+    /// Get the path to the source code on the host machine (outside the sandbox).
+    pub fn source_dir(&self) -> PathBuf {
```

Comment:
Why was this method changed to be `pub`?

Reply:
it is needed by docs.rs, see the linked PR

Reply:
That wouldn't work, see https://github.com/rust-lang/docs.rs/pull/1193#discussion_r530415987.